core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -264,68 +265,6 @@ class ReplicaManagerTest {
     when(cache.getAliveBrokerNodes(any[ListenerName])).thenReturn(aliveBrokers)
   }
 
-  @Test
-  def testMaybeAddLogDirFetchersWithoutEpochCache(): Unit = {
```

Comment:
@junrao do you agree that we don't need this test anymore? Or is there a purpose that is not made clear by the test name?

Reply:
This test is still useful. We can just leave `partition.futureLog.get.leaderEpochCache` unchanged. It will be initialized as empty and the fetcher thread will default to initializing the fetch offset with HWM. We probably want to adjust the test name accordingly.